src/services/jsDoc.ts

```diff
@@ -1,48 +1,92 @@
 /* @internal */
 namespace ts.JsDoc {
     const jsDocTagNames = [
+        "abstract",
+        "access",
+        "alias",
+        "arg",
+        "argument",
+        "async",
         "augments",
         "author",
-        "argument",
         "borrows",
         "callback",
         "class",
+        "classDesc",
+        "const",
         "constant",
         "constructor",
         "constructs",
+        "copyright",
         "default",
         "deprecated",
         "description",
+        "emits",
+        "enum",
         "event",
         "example",
+        "exports",
         "extends",
+        "external",
         "field",
+        "file",
         "fileOverview",
+        "fires",
+        "func",
         "function",
+        "generator",
+        "global",
+        "hideConstructor",
+        "host",
         "ignore",
+        "implements",
         "inheritDoc",
         "inner",
+        "instance",
+        "interface",
+        "kind",
         "lends",
+        "license",
         "link",
+        "linkCode",
+        "linkPlain",
+        "listens",
+        "member",
         "memberOf",
         "method",
+        "mixes",
+        "module",
         "name",
         "namespace",
+        "override",
+        "package",
         "param",
         "private",
         "prop",
         "property",
+        "protected",
         "public",
+        "readonly",
         "requires",
+        "return",
         "returns",
         "see",
         "since",
         "static",
+        "summary",
         "template",
+        "this",
         "throws",
+        "todo",
+        "tutorial",
         "type",
         "typedef",
-        "version"
+        "var",
+        "variation",
+        "version",
+        "virtual",
+        "yield",
+        "yields"

```

Comment:
do we need both `@yeild` and `@yeilds`?

Reply:
They are both part of the jsdoc tag names list, it's the same question as for `return` and `returns`, `prop` and `property`, `const` and `constant`, `desc` and `description` and so on, they are defined as synonyms for convenience.

Reply:
The doc generator understands them for convenience, but I think for providing completion we should just stick with `yields`.